benchmark/probes/benchmark.bpf.h

```diff
@@ -3,15 +3,15 @@
 #include <bpf/bpf_tracing.h>
 
 #if defined(__TARGET_ARCH_x86)
-#define FENTRY_SEC() SEC("fentry/__x64_sys_getpid")
+#define FENTRY_SEC() SEC("fentry/__do_sys_getpid")
```

Comment:
I can see both on our x86_64 running v6.1:

```
$ cat /proc/kallsyms | grep -E '( __x64_sys_getpid|__do_sys_getpid)$'
0000000000000000 t __do_sys_getpid
0000000000000000 T __x64_sys_getpid
```

On arm64 we only have one:

```
$ cat /proc/kallsyms | grep -E '( __arm64_sys_getpid|__do_sys_getpid)$'
0000000000000000 T __arm64_sys_getpid
```

Switching to `__do_sys_getpid` makes sense given what you say, even if it makes me sad for the lack of consistency.